From 81593390d63393f14f2d41523797636bdad19ae5 Mon Sep 17 00:00:00 2001
From: Michael Jeanson <mjeanson@efficios.com>
Date: Thu, 12 Nov 2015 11:27:47 -0500
Subject: [PATCH] Fix build on non-Linux Debian ports

Signed-off-by: Michael Jeanson <mjeanson@efficios.com>
Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
---
 urcu/syscall-compat.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/urcu/syscall-compat.h b/urcu/syscall-compat.h
index 288cfaf..0e9f26c 100644
--- a/urcu/syscall-compat.h
+++ b/urcu/syscall-compat.h
@@ -29,7 +29,7 @@
 
 #if defined(__ANDROID__) || defined(__sun__)
 #include <sys/syscall.h>
-#elif defined(__linux__)
+#elif defined(__linux__) || defined(__GLIBC__)
 #include <syscall.h>
 #else
 #error "Add platform support to urcu/syscall-compat.h"
