[PATCH] Add hppa architecture which uses gcc atomic builtins

Signed-off-by: Helge Deller <deller@gmx.de>

diff -up ./configure.ac.org ./configure.ac
--- ./configure.ac.org	2015-02-02 15:38:51.787316448 +0100
+++ ./configure.ac	2015-02-02 15:46:38.387440069 +0100
@@ -81,6 +81,7 @@ AS_CASE([$host_cpu],
 	[aarch64], [ARCHTYPE="gcc"],
 	[mips*], [ARCHTYPE="mips"],
 	[tile*], [ARCHTYPE="gcc"],
+	[hppa*], [ARCHTYPE="gcc"],
 	[ARCHTYPE="unknown"]
 )
 
